iosApp/iosApp/Modules/Dashboard/Dashboard.storyboard

```diff
@@ -87,64 +87,33 @@
                                                                     <nil key="textColor"/>
                                                                     <nil key="highlightedColor"/>
                                                                 </label>
                                                             </subviews>
                                                         </stackView>
-                                                        <view contentMode="scaleToFill" horizontalCompressionResistancePriority="1000" translatesAutoresizingMaskIntoConstraints="NO" id="OuC-gA-4kk">
-                                                            <rect key="frame" x="299" y="0.0" width="12" height="202"/>
-                                                            <subviews>
-                                                                <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="qHp-Ta-IwU">
-                                                                    <rect key="frame" x="0.0" y="0.0" width="12" height="12"/>
-                                                                    <constraints>
-                                                                        <constraint firstAttribute="height" constant="12" id="ehF-hF-JXY"/>
-                                                                        <constraint firstAttribute="width" constant="12" id="uBP-zm-has"/>
-                                                                    </constraints>
-                                                                    <state key="normal" title="Button"/>
-                                                                    <buttonConfiguration key="configuration" style="plain"/>
-                                                                </button>
-                                                            </subviews>
-                                                            <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-                                                            <constraints>
-                                                                <constraint firstAttribute="trailing" secondItem="qHp-Ta-IwU" secondAttribute="trailing" id="5oF-tt-k9S"/>
-                                                                <constraint firstItem="qHp-Ta-IwU" firstAttribute="top" secondItem="OuC-gA-4kk" secondAttribute="top" id="ruf-Tk-gEb"/>
-                                                                <constraint firstItem="qHp-Ta-IwU" firstAttribute="leading" secondItem="OuC-gA-4kk" secondAttribute="leading" id="xW7-wZ-OGe"/>
-                                                            </constraints>
-                                                        </view>
```

Comment:
You can set stack view alignment to top instead of add extra container view. This will stop stretching the image